FileShareAdminClient/Models/Response/Result.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using UKHO.FileShareClient.Internal;
+
+namespace UKHO.FileShareAdminClient.Models.Response
+{
+    public class Result<T> : IResult<T>
+    {
+        public bool IsSuccess { get; set; }
+
+        public int StatusCode { get; set; }
+
+        public List<Error> Errors { get; set; } = new List<Error>();
+
+        public T Data { get; set; }
+
+        /// <summary>
+        /// Deserialize the response data.
+        /// </summary>
+        /// <param name="successCode">HttpStatusCode</param>
+        /// <param name="response">API response</param>
+        /// <returns></returns>
+        public async Task ProcessHttpResponse(HttpStatusCode successCode, HttpResponseMessage response)
```

Comment:
Please change `ProcessHttpResponse` to `internal`

Reply:
Done the change as you specified.